packages/react-dom/package.json

```diff
@@ -63,9 +63,9 @@
       "bun": "./server.bun.js",
       "deno": "./server.browser.js",
       "worker": "./server.browser.js",
-      "browser": "./server.browser.js",
       "node": "./server.node.js",
       "edge-light": "./server.edge.js",
+      "browser": "./server.browser.js",
```

Comment:
I just want to add one concern: this makes it very hard to use the actual browser build in tests when e.g. running the node test runner - `node` always runs with the `node` condition, and there can only be additional ones added, but `node` itself cannot be removed.

So previously it was possible to run the browser build of React in a node testing environment with `--condition=browser`, and after this it will be impossible.

I'm gonna say: I'm not sure if it's relevant, but I wanted to add the thought to the conversation.

Reply:
I think the solution is to use mocking like
```
jest.mock('react-dom/server', () => require('react-dom/server.browser'))
```